### ISSUE TITLE
Earthquake Prediction Model using Python _Phase 1& Phase 2

### ISSUE THREAD

**Issue by VjRasheed** [VjRasheed/Earthquake#1]
Earthquake Prediction Using Python
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.ensemble import RandomForestClassifier
from sklearn.metrics import accuracy_score

data = pd.read_csv('earthquake_data.csv')

x = data.drop('earthquake_occured',axis=1)
y = data['earthquake_occured']

x_train,x_test,y_train,y_test = train_test_split(x,y,test_size=0.2,random_state=42)
clf = RandomForestClassifier()

clf.fit(x_train,y_train)

y_pred = clf.predict(x_test)

accuracy = accuracy_score(y_test,y_pred)
print(f"Accuracy:{accuracy}")

**Comment by VjRasheed** [comment]
![Image](https://github.com/VjRasheed/Earthquake/assets/146167975/255a8343-20b0-4387-9c13-d1bcef793376)
